server/models/Submission.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+
+const submissionSchema = new Schema({
+  url: {
+    type: String,
+    required: true
+  },
+  creator: {
+    type: Schema.Types.ObjectId,
+    ref: "User",
+    required: true
+  },
+  contest: {
+    type: Schema.Types.ObjectId,
+    ref: "Contest",
+    required: true
+  }
+});
+
```

Comment:
Here a dateCreated might be nice so that when listing all submissions for a contest, the contest maker can order by submission date. For versioning, we can keep a submissionVersion[ creator (PK), contest (PK), versionNumber, url] as a weak entity while we keep the url in submission model to be latest so we can easily access it without joins. But that's for later